pkgs/applications/misc/gnuradio-nacl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, boost, gnuradio, uhd
+, makeWrapper, libsodium, cppunit
+, pythonSupport ? true, python, swig
+}:
+
+assert pythonSupport -> python != null && swig != null;
+
+stdenv.mkDerivation rec {
+  name = "gnuradio-nacl-${version}";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "stwunsch";
+    repo = "gr-nacl";
+    rev = "d6dd3c02dcda3f601979908b61b1595476f6bf95";
+    sha256 = "0q28lgkndcw9921hm6cw5ilxd83n65hjajwl78j50mh6yc3bim35";
+  };
+
+  buildInputs = [
+    cmake pkgconfig boost gnuradio uhd makeWrapper libsodium cppunit
+  ] ++ stdenv.lib.optionals pythonSupport [ python swig ];
+
+  postInstall = ''
+    for prog in "$out"/bin/*; do
+        wrapProgram "$prog" --set PYTHONPATH $PYTHONPATH:$(toPythonPath "$out")
+    done
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Gnuradio block for encryption";
+    homepage = https://github.com/stwunsch/gr-nacl;
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+    maintainers = with maintainers;  [ bjornfor the-kenny ];
```

Comment:
Vladimír Čunát <notifications@github.com> writes:
>
> Do these people know they will become maintainers if this package?
>
No.  they are maintainers of other gnuradio bits.  I fixed it and pushed
new version now

-- 
Matthew O'Gorman
BM-NBUmia4p88Jny5bZbRGcMt64SWWp5WVv
TorChat: 5w3dtk7nhkkijcpc
Site: https://b.rldn.net
Xim/Email: mog@rldn.net
mog@rldn:~$ fortune wisdom -s|cowsay -f /dev/null
 ____________________________________ 
/ No matter where I go, the place is \
\ always called "here".              /
 ------------------------------------